docs/user/build/install/jdk.rst

```diff
@@ -48,6 +48,24 @@ Java 5    GeoTools 2.5.x   GeoTools 8.x     compiler=1.5
 Java 1.4  GeoTools 2.x     GeoTools 2.4.x   compiler=1.4 
 ========= ================ ================ =================
 
+Using Java 9 with GeoTools
+''''''''''''''''''''''''''
+
+Java 9 introduces a number of changes to the JVM, most notably the module system (Project Jigsaw). Refer to `The State of the Module System <http://openjdk.java.net/projects/jigsaw/spec/sotms/>`_ for more details.
+While GeoTools has not yet been updated to use the module system, GeoTools 19.x can run on Java 9.
+
+If your project depends on GeoTools 19.x, and you want to use it with Java 9, you will need to add the following flags to your JVM runtime arguments::
+
+    --add-modules=java.xml.bind --add-modules=java.activation
```

Comment:
Wondering, this makes it look like the issue is GT wide, but I thought it was a problem specific to image mosaic and modules depending on it?

Reply:
This is an issue wherever JAXB and Activation are used. The first place I encountered it was in imagemosaic, so anything that comes before it in the build order should be unaffected. However, since I was mostly addressing issues as they came up in build order, I'm not sure if there are any latter modules that share this issue (For example, it is possible that gt-render also requires this flag).

So, it probably just affects imagemosaic and dependants, but I don't know enough to say for certain; consequently I decided to err on the side of caution.